### PR TITLE
feat(codemod): elasticsearch migrations v8

### LINKED ISSUE #827: [codemod][new] elastic javascript client - migration to v8.14
### Codemod description
As described in the [upgrade guide](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/7.17/breaking-changes.html):
"The returned value of an API call will no longer be the body, statusCode, and headers for callbacks, and only the body for promises. The new returned value will be a unique object containing the body, statusCode, headers, warnings, and meta, for both callback and promises."

----
To contributors working on this task:
- [ ] **Subject Matter/Framework Expert**:  Share the Codemod Studio link (example) with detailed specs from the framework expert, including before/after pairs and transformation logic, along with edge cases.
- [ ] **Codemod Expert**: [Build](https://go.codemod.com/build-codemod-docs) and [publish](https://go.codemod.com/codemod-publish-doc) quality codemods that cover most edge cases (false negatives are acceptable) with no false positives.
- Questions? -> [Community](https://go.codemod.com/community)

### PR DESCRIPTION
**Pull request by 35C4n0r**
#### 📚 Description
https://codemod.com/registry?q=elasticsearch

#### 🔗 Linked Issue
Closes #827

### CODE REVIEW

File: packages/codemods/elasticsearch/8/migrate-plugins-extend/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@codemod/elasticsearch-8-migrate-plugins-extended",
+  "dependencies": {},
+  "devDependencies": {
```
Comment (mohebifar) [inline packages/codemods/elasticsearch/8/migrate-plugins-extend/package.json:4]:
```json
"private": true
```

File: packages/codemods/elasticsearch/8/migration-recipe/package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@codemod/elasticsearch-8-migration-recipe",
+  "files": ["./README.md", "./.codemodrc.json"],
+  "type": "module"
```
Comment (mohebifar) [inline packages/codemods/elasticsearch/8/migration-recipe/package.json:4]:
```json
"private": true
```

File: packages/codemods/elasticsearch/8/move-client-configurations/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@codemod/elasticsearch-8-move-client-configurations",
+  "dependencies": {},
```
Comment (mohebifar) [inline packages/codemods/elasticsearch/8/move-client-configurations/package.json:3]:
```json
"private": true
```

File: packages/codemods/elasticsearch/8/rename-query-string/package.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@codemod/elasticsearch-8-rename-query-string",
+  "dependencies": {},
```
Comment (mohebifar) [inline packages/codemods/elasticsearch/8/rename-query-string/package.json:3]:
```json
"private": true
```

File: packages/codemods/elasticsearch/8/unify-return-value/package.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@codemod/elasticsearch-8-unify-return-value",
+  "dependencies": {},
```
Comment (mohebifar) [inline packages/codemods/elasticsearch/8/unify-return-value/package.json:3]:
```json
"private": true
```

**Review by mohebifar — CHANGES_REQUESTED** [review]
Please make sure to add 

```json
"private": true
```

to packages that are not published to npm

**Comment by 35C4n0r** [comment]
@mohebifar thanks for the review :)
Made the changes.

**Comment by mohab-sameh** [comment]
@35C4n0r Do you mind moving this to [`codemod-com/commons`](https://github.com/codemod-com/commons) as we're moving codemod package sources there? For now, I will close this PR.

**Pull request closed without merging**